eq-author-api/utils/datastoreDynamo.js

```diff
@@ -56,7 +56,7 @@ const getQuestionnaire = id => {
   });
 };
 
-const MAX_UPDATE_TIMES = 2;
+const MAX_UPDATE_TIMES = 5;
```

Comment:
Why is this necessary?
Does removing the equal fix solve the issue?

Reply:
Unsure at the moment.  Having cypress issues and giving a few things a go!

Reply:
If the isEqual is to blame. Maybe move it to a separate PR?

Reply:
There's UI errors as well 😕 
<img width="1392" alt="screenshot 2019-03-08 at 15 59 20" src="https://user-images.githubusercontent.com/2471296/54039601-4000e100-41bb-11e9-8aa4-d644f18f44b5.png">

Reply:
Weird.